Wsl.psm1

```diff
@@ -272,6 +272,34 @@ function Get-WslDistribution
     }
 }
 
+<#
+.EXTERNALHELP
+Wsl-help.xml
+#>
+# Retrieves listing of distributions available from online sources
+function Get-WslDistributionOnline {
+    [CmdletBinding()]
+    param(
+    )
+
+    process {
+        $store = $false
+        Invoke-Wsl "--list", "--online" -IgnoreErrors | Select-Object -Skip 1 | ForEach-Object {
+            $name, $friendlyName = $_ -split ' ', 2
+            if ($store -eq $true ) {
+                $friendlyName = $($friendlyName.Split(@(" "), [System.StringSplitOptions]::RemoveEmptyEntries)) -join " "

```

Comment:
```suggestion
                $friendlyName = $friendlyName.Trim()
```

Reply:
I will do some further testing with this.  When I tried doing this with Trim() previously, it did not remove the excess spaces at the start of the string effectively, and is why I resorted to the convoluted way in the current code, which removes all spaces and then reinserts them back in.

Reply:
Trim() is also working correctly now.  This could be as a result of implementing the "-split" a few lines earlier as once I put this in did not go back and do more testing with the convoluted call.